contracts/oracle/Oracle.sol

```diff
@@ -179,7 +179,11 @@ contract Oracle is RoleModule {
     // @param params OracleUtils.SetPricesParams
     function _setPrices(
         OracleUtils.ValidatedPrice[] memory prices
-    ) internal returns (OracleUtils.ValidatedPrice[] memory) {
+    ) internal {
+        if (prices.length == 0) {
```

Comment:
when will prices.length be zero? it may be useful to add a comment here with an example of this case

Reply:
added comment